x/staking/migrations/v046/store.go

```diff
@@ -19,6 +19,10 @@ func MigrateStore(ctx sdk.Context, storeKey storetypes.StoreKey, cdc codec.Binar
 }
 
 func migrateParamsStore(ctx sdk.Context, paramstore paramtypes.Subspace) {
-	paramstore.WithKeyTable(types.ParamKeyTable())
-	paramstore.Set(ctx, types.KeyMinCommissionRate, types.DefaultMinCommissionRate)
+	if paramstore.HasKeyTable() {
```

Comment:
encountered the same bug as in https://github.com/cosmos/cosmos-sdk/pull/9484, so applied the same fix